### DISCUSSION TITLE
Form Block - Expanding size of data entry fields

### DISCUSSION THREAD

**Discussion by alansinger** [Budibase/budibase#8922] [Help] [answered]
I have an auto-generated edit screen for a table and I want one of the data entry fields to be able to show 3 rows of text - the default size in the Form Block is one row.
Can anyone suggest how I can achieve this pls?

**Comment by melohagan** [comment, marked as answer]
Hey @alansinger 

You could try using the [Long form field](https://docs.budibase.com/docs/long-form-field). You will need to update the data-type in the Budibase table, like so:

![Screenshot 2022-12-07 at 15 41 36](https://user-images.githubusercontent.com/101575380/206223839-a1596b7f-f518-4d5d-9174-943b8d17c21b.png)

![Screenshot 2022-12-07 at 15 42 05](https://user-images.githubusercontent.com/101575380/206223962-97847933-060b-430f-99a4-13c6b75cc339.png)

It's about 7 lines before it adds a scrollbar. 

However you can specify line height. It does seem to have a min-height of three lines which is convenient!
![Screenshot 2022-12-07 at 15 50 14](https://user-images.githubusercontent.com/101575380/206226050-857594cf-389c-4198-9629-b836ef5330ad.png)

![Screenshot 2022-12-07 at 15 51 46](https://user-images.githubusercontent.com/101575380/206226376-251de1bc-84d2-45e0-997e-35419e7aefc5.png)

**Reply by melohagan** [reply]
You will also need to **Eject Block** to be able to add the `Long form field`
